src/applications/MachineShopSimulator.java

```diff
@@ -11,36 +11,12 @@
     public static final String BAD_MACHINE_NUMBER_OR_TASK_TIME = "bad machine number or task time";
 
     // data members of MachineShopSimulator
-    private int timeNow; // current time
+    int timeNow; // current time

```

Comment:
Good job making this private!

Reply:
I was not paying attention here, and gave you credit for making that field `private`, when you did the exact _opposite_. That's arguably a Bad Thing (you want _more_ encapsulation rather than less), so I should have been grumpy rather than enthusiastic.